.github/workflows/scala.yml

```
@@ -29,3 +27,20 @@ jobs:
         cache: 'sbt'
     - name: Run tests
       run: sbt test
+    - name: Cache sbt dependencies
+      uses: actions/cache@v1
```

actionlint

the runner of "actions/cache@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)